### ISSUE TITLE
Wordpress 404 errors with Permalinks set to Post Name

### ISSUE THREAD

**Issue by gerrymiller** [widdix/aws-cf-templates#111]
Nice work on this template! I am getting 404 errors when setting Permalinks to "Post Name," rather than the default "Plain." Any ideas?

**Comment by andreaswittig** [comment]
@gerrymiller Which template are you using `wordpress-ha.yaml` or `wordpress-ha-aurora.yaml`?

**Comment by gerrymiller** [comment]
Hi @andreaswittig I am using wordpress-ha-aurora.yaml.

**Comment by gerrymiller** [comment]
One more piece of information - although the home page still comes up, I believe that's because it's cached in CloudFront. It appears all healthchecks are failing as well, as the ASG continues to kill instances and spin new ones up until I configure WP back to Plain Permalinks.

**Comment by ILLUMINICE** [comment]
Hi Gerry,

Please see the most recent post in my Issue #110 

It looks like you and I are having the very same problem, though my path to discovering it was different from yours.

I hope @andreaswittig can help us to resolve this...

All my best!

~ Michael

**Comment by gerrymiller** [comment]
I believe I solved this problem. In the end it was a permissions issue with `.htaccess`; I had to manually go to `/var/www/html` and

```
sudo touch .htaccess
sudo chown apache:apache .htaccess
```

After that, when I set Permalinks to Post Name, Wordpress was able to update `.htaccess`. I recommend adding those lines to the CF template (I didn't do this as I had already deployed and made updates to the EFS files).

I still had the issue described in #110 which I believe I solved as well and will annotate in that issue.
